### DISCUSSION TITLE
Welcome to freemocap Discussions!

### DISCUSSION THREAD

**Discussion by jonmatthis** [freemocap/freemocap#51] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rphad23** [comment]
i have this error:
    _message.Message._CheckCalledFromGeneratedFile()
TypeError: Descriptors cannot not be created directly.
If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
If you cannot immediately regenerate your protos, some other possible workarounds are:
 1. Downgrade the protobuf package to 3.20.x or lower.
 2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).
how do I fix it?

**Reply by ghost** [reply]
Posted by @jrem090:

While in the freemocap-env:
pip install protobuf==3.19

This should uninstall the current version and give you the latest minor version of 3.19
